psi4/src/psi4/libfock/cubature.cc

```diff
@@ -4564,15 +4579,17 @@ void MolecularGrid::print_details(std::string out, int /*print*/) const {
     std::shared_ptr<psi::PsiOutStream> printer = (out == "outfile" ? outfile : std::make_shared<PsiOutStream>(out));
     printer->Printf("   > Grid Details <\n\n");
     for (size_t A = 0; A < radial_grids_.size(); A++) {
-        printer->Printf("    Atom: %4zu, Nrad = %6d, Alpha = %11.3E:\n", A, radial_grids_[A]->npoints(),
-                        radial_grids_[A]->alpha());
-        for (size_t R = 0; R < spherical_grids_[A].size(); R++) {
-            double Rval = radial_grids_[A]->r()[R];
-            double Wval = radial_grids_[A]->w()[R];
-            int Nsphere = spherical_grids_[A][R]->npoints();
-            int Lsphere = spherical_grids_[A][R]->order();
-            printer->Printf("    Node: %4zu, R = %11.3E, WR = %11.3E, Nsphere = %6d, Lsphere = %6d\n", R, Rval, Wval,
-                            Nsphere, Lsphere);
+        printer->Printf("    Atom: %4zu, Nrad = %6d, Alpha = %11.3E:\n", A,
```

Comment:
Just to be clear on this point: `radial_grids_` exists now as a convenience struct for the sake of this printout, but the data being printed actually existed and is used elsewhere. Is that right, or am I missing something?

Reply:
Yes, `radial_grids_` is a convenience struct. I checked, and it looks like, e.g., `r` and `wr` *do* get used downstream, or at least, used in the construction of the `MassPoint`s that get `push_back`'d into the `atomic_grids_` of `MolecularGrid`. I suppose I haven't checked what happens, long-term, with those, but hopefully something...

Reply:
Okay, in that case, does it make more sense to not print this at all but rely on the user to call `MassPoint.print()` or `MassPoint.print_details()` when necessary?

Reply:
I looked into this and my conclusion was that grid _pruning_ also acts on the `MassPoint`s, so recovering things like the initial lebedev grid or number of grid points isn't possible. The `MassPoint`s don't really carry around sufficient information to reconstruct the state printed in `print_details`. But I'm not very familiar with the codebase OR what's happening under-the-hood here, so another set of eyes to check my assertion that this Cannot Be Done would be appreciated.

However, if ya'll are OK ditching `print_details` all together, then all these problems go away. The replacement structs were added b/c of @susilehtola 's requests to not break the existing print information.

Reply:
Any information that isn't actually used except in printing, I'm okay removing.